openapi/index.yaml

```diff
@@ -1,4 +1,4 @@
-swagger: '2.0'
+swagger: "2.0"
 info:
   version: 1.0.0
```

Comment:
We need to update the version here.

Reply:
Fixed in [469e173](https://github.com/pagopa/io-functions-services/pull/67/commits/469e173201722102ed04c60b05300f6618a02fe2)

Reply:
The comment was referring to the "version" field not the swagger version.
We are adding new operations so we can update the version from 1.0.0 to something like 1.1.0.

Reply:
Fixed in [eba6c35](https://github.com/pagopa/io-functions-services/pull/67/commits/eba6c353f5eeb25ccbc5b29f7fcfdb4b724b85bd)